honeybadger/payload.py

```diff
@@ -94,15 +95,37 @@ def create_payload(exception, exc_traceback=None, config=None, context={}):
     if exc_traceback is None:
         exc_traceback = sys.exc_info()[2]
 
-    payload = default_plugin_manager.generate_payload(config, context)
+    request_payload = default_plugin_manager.generate_payload(config, context)
 
-    return {
+    payload =  {
         'notifier': {
             'name': "Honeybadger for Python",
             'url': "https://github.com/honeybadger-io/honeybadger-python",
             'version': __version__
         },
         'error':  error_payload(exception, exc_traceback, config),
         'server': server_payload(config),
-        'request': payload,
+        'request': request_payload,
+        'details': {}
     }
+
+    if config.is_aws_lambda_environment:
+        AWS_ENV_MAP = [
```

Comment:
@stympy do you think we should normalize this data like [in the Ruby gem](https://github.com/honeybadger-io/honeybadger-ruby/blob/3c8c4ff191ba39ca77305d129b004826a08bb919/lib/honeybadger/util/lambda.rb#L5) (convert the key names)?

Reply:
yes, I think that makes sense